app/views/admin/calendar/_legend.slim

```diff
@@ -0,0 +1,54 @@
+div#legend-content
+  div.cal-legend
+    div.row
+      div.column.col-lg-6
+        table.legend-block
+          tr
+            td
+              i.fa.fa-calendar-check-o
```

Comment:
Please remember the `=` shortcut to make things like this more compact. i.e. `td = i.fa.fa-calendar-check-o`